docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Execute the `run_sscan()` plan using the bluesky RunEngine `RE`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Started scan ...\n",
+      "Scan done.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the data from the `sscan` record, we use `ophyd.EpicsSignalRO` to connect with the arrays.  We'll do that in a later section of this notebook.\n",
+    "\n",
+    "### 1.B. wait with a Status object\n",
+    "\n",
+    "Let's eliminate that polling loop.  This will make the plan respond more quickly\n",
+    "when the button changes value.  The `ophyd.status.Status` class (runs in the\n",
+    "background) reports when the scan has finished or times out.  We define an inner\n",
+    "`watch_the_button()` function to detect when the button goes from `1` to `0`\n",
+    "(the scan stops).  We setup a subscription (*after* we start the move) to\n",
+    "monitor the button value and respond immediately.  \n",
+    "\n",
+    "We wait for the status object to finish with `st.wait()`.  In a bluesky plan,\n",
+    "we'll need to use `apstools.plans.run_blocking_function` with `st.wait()` since\n",
+    "it is a blocking function.\n",
+    "\n",
+    "Finally, we cancel the monitor of the button."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from apstools.plans import run_blocking_function\n",
+    "from ophyd.status import Status\n",
+    "\n",
+    "def run_sscan():\n",
+    "    st = Status(timeout=20)\n",
+    "\n",
+    "    def watch_the_button(old_value, value, **kwargs):\n",
+    "        if old_value == 1 and value == 0:\n",
+    "            st.set_finished()\n",
+    "            scan_button.clear_sub(watch_the_button)\n",
+    "\n",
+    "    yield from bps.mv(scan_button, 1)\n",
+    "    scan_button.subscribe(watch_the_button)\n",
+    "    yield from run_blocking_function(st.wait)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Run preconfigured `sscan` record\n",
+    "\n",
+    "The `apstools.synApps.SscanRecord` class provides access to most of the fields\n",
```

Comment:
I am not sure I understand what was the point of the first section. Why not skip it to use the `apstools.synApps.SscanRecord` directly?

Reply:
Otherwise I have the same questions as above for the code.

Reply:
The first section is because I am familiar with coding practices at beamlines.  Most common technique to wait is to use a polling loop in the main code.  The first section is exactly that practice.

Was thinking that this particular comparison (polling loop v. `status.wait()`) would be its own (very short) howto.  It has some general use.  Agree?  If so, we could skip the first section and refer to the polling/status howto for background.